react-webpack/README.md

```diff
@@ -0,0 +1,18 @@
+# React, webpack
+
+Full documentation about it [here](https://mochajs.org/#-require-module-r-module)
+
+## Running the application
+
+Run `npm run build` (see below) and then open `index.html` inside a browser.
+
+## Commands
+
+- `npm run build` - run webpack using the local `webpack.config.js` file. Builds the client-side code into `/lib`.
+- `npm run test` - run the tests using the local `.mocharc.js` config file. As the config includes the Babel transpilation hook `@babel/register` it does not require pre-compilation before running.
+
+## Makes use of
+
+- [babel](https://babeljs.io/) - transpilation
+- [enzyme](https://airbnb.io/enzyme/) - React component testing library
+- [chai](https://www.chaijs.com/api/assert/) - assertion library
```

Comment:
Consider adding JSX src and test files to the mix

Reply:
Wasnt sure what you meant here, added a link for jsx, let me know if thats not right.

Reply:
Not sure what was unclear -- create two JSX files, one src and one test.

Reply:
Sorry im still confused. I have jsx _in_ both my src files, and a small bit of jsx in my test. Could you explain a bit more please?

Reply:
I contributed to that confusion, I'm sure (no JSX experience). Rephrased, files containing JSX content should have ".jsx" extension. Having done recent user troubleshooting on something similar, would like example to have _both_ "pure" JS file _as well as_ JSX file. This will need Mocha's "extension" option to be `["js", "jsx"]`, I believe. Nice if we could also have "test/**/*.spec.js[x]" (again, one of each).

Reply:
AFAIK the Facebook team itself is keen that `js` should be the only extension no matter what is in the file https://github.com/facebook/create-react-app/issues/87#issuecomment-234627904. I have not seen a jsx extension in a long while personally. 
I feel if we are aiming for best practices we should keep away from a `.jsx` extension.
Thoughts?